docs/reference/docs/index_.asciidoc

```diff
@@ -39,22 +152,9 @@ The result of the above index operation is:
 --------------------------------------------------
 // TESTRESPONSE[s/"successful" : 2/"successful" : 1/]
 
-The `_shards` header provides information about the replication process of the index operation:
-
-`total`:: Indicates how many shard copies (primary and replica shards) the index operation should be executed on.
-`successful`:: Indicates the number of shard copies the index operation succeeded on.
-`failed`:: An array that contains replication-related errors in the case an index operation failed on a replica shard.
-
-The index operation is successful in the case `successful` is at least 1.
-
-NOTE:   Replica shards may not all be started when an indexing operation successfully returns (by default, only the
-        primary is required, but this behavior can be <<index-wait-for-active-shards,changed>>). In that case,
-        `total` will be equal to the total shards based on the `number_of_replicas` setting and `successful` will be
-        equal to the number of shards started (primary plus replicas). If there were no failures, the `failed` will be 0.
-
 [float]
 [[index-creation]]
-==== Automatic Index Creation
+==== Automatic index creation
```

Comment:
I would consider moving this and the following sections into a long-form [`Description` section](https://github.com/elastic/docs/blame/master/shared/api-ref-ex.asciidoc#L39).

Some of these sections seem to be explaining functionality vs. providing examples.

Reply:
Done.